drpcmetadata/proto/metadata.proto

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "proto";
+
+package metadata;
+
+message InvokeMetadata {
+    uint32 version = 1;
```

Comment:
We don't need this field because protobuf's wire format allows us to add fields in the future.